Sample Data/Perimeter81_ActivityLogs_sample.csv

```diff
@@ -0,0 +1,18 @@
+TimeGenerated,"installation_installationId_g","error_message_s","application_name_s","application_type_s","application_endpoint_s","application_alias_cname_s","network_name_s","network_dns_s","releasedFrom_name_s","releasedFrom_company_s","releasedBy_tenantId_s","releasedBy_lastName_s","releasedBy_firstName_s","eventName_s","tenantId_s","originalTenantId_s","releasedFrom_tenantId_s","releasedBy_email_s","ip_s"
```

Comment:
Making the sample data richer enables the Azure Sentinel community to use this sample data and build richer experiences for your integration in Azure Sentinel which adds more breadth to the current integration. Hence would be great to take care of the following. 
1. In your activity logs for eventName_s do you have only "api.activity.login.fail" and "api.activity.application.session.start" type values? Aren't there any other eventName_s? If yes, please include those types in sample data as well.
2. Are there other types of application_type_s values you can include in the sample data?
3. Are there other values of error_message_s that can be added to the sample data?
4. Are the activity logs that  you are surfacing in Azure Sentinel at parity in terms of what's being surfaced in Perimeter 81 product? If no, what are the differences and is it possible to include that data as part of this integration as well?

Reply:
1. Added more events.
2. Added another application type with a total of 2.
3. Added more error messages.
4. Yes, it is the same.